packages/salesforcedx-vscode-lwc/test/vscode-integration/metaSupport/metaSupport.test.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { extensions, window } from 'vscode';
+import { nls } from '../../../src/messages';
+import { MockRedhatExtension } from './MockRhExtension';
+import { metaSupport } from '../../../src/metasupport';
+var assert = require('assert');
```

Comment:
Let's change this `require` to an `import` since the current format breaks the use of bundling tools like webpack which we're currently introducing in the project.